src/test/java/api/loans/DeclareLostAPITests.java

```diff
@@ -231,6 +237,104 @@ void shouldChargeProcessingAndItemFeesWhenBothDefined() {
       hasJsonPath("feeFineType", "Lost item processing fee"),
       hasJsonPath("amount", expectedProcessingFee),
       hasJsonPath("contributors[0].name", contributorName)));
+
+    assertNull(actualCostRecord);
+  }
+
+  @Test
+  void shouldChargeProcessingFeesOnlyWhenBothDefinedForActualCostCharge() {
```

Comment:
The test name is a bit confusing. We should check that processing fee is NOT charged here maybe? And change the name to shouldOnlyCharge

Reply:
Ok, changed. Thanks